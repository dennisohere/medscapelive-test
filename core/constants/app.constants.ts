export const APP_SCRIPT_COMPLETE = 'complete'
export const VIRTUAL_EVENT_PLATFORM_MEDSCAPE = 'Medscape'
export const EVENT_PRESENCE_QUERY_PARAM = 'eventPresence'
export const INCLUDE_MNG_EVENTS = true
