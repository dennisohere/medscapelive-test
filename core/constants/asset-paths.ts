export const X2 = `2x`
export const X3 = `3x`

export const HOME_BLUE_HERO_HEADER = `/static_vue/medscape-live-web/img/home-blue-hero-header/header.jpg`
export const HOME_BLUE_HERO_HEADER_SRCSET = `/static_vue/medscape-live-web/img/home-blue-hero-header/header.jpg, /static_vue/medscape-live-web/img/home-blue-hero-header/header@2x.jpg 2x, /static_vue/medscape-live-web/img/home-blue-hero-header/header@3x.jpg 3x`

export const HOME_MARKETING_CARD1 = `/static_vue/medscape-live-web/img/home-b2c-marketing/card1.png`
export const HOME_MARKETING_CARD2 = `/static_vue/medscape-live-web/img/home-b2c-marketing/card2.jpg`
export const HOME_MARKETING_CARD3 = `/static_vue/medscape-live-web/img/home-b2c-marketing/card3.png`

export const LIVE_EVENT_PLACEHOLDER_IMG1 = `/static_vue/medscape-live-web/img/live-event-placeholder-img/default1.png`
export const LIVE_EVENT_PLACEHOLDER_IMG2 = `/static_vue/medscape-live-web/img/live-event-placeholder-img/default2.png`
export const LIVE_EVENT_PLACEHOLDER_IMG3 = `/static_vue/medscape-live-web/img/live-event-placeholder-img/default3.png`

export const MNG_EVENT_PLACEHOLDER_IMG1 = `/static_vue/medscape-live-web/img/mng-event-placeholder-img/img1.png`
export const MNG_EVENT_PLACEHOLDER_IMG2 = `/static_vue/medscape-live-web/img/mng-event-placeholder-img/img2.png`
export const MNG_EVENT_PLACEHOLDER_IMG3 = `/static_vue/medscape-live-web/img/mng-event-placeholder-img/img3.png`
export const MNG_EVENT_PLACEHOLDER_IMG4 = `/static_vue/medscape-live-web/img/mng-event-placeholder-img/img4.png`
export const MNG_EVENT_PLACEHOLDER_IMG5 = `/static_vue/medscape-live-web/img/mng-event-placeholder-img/img5.png`

export const MEDSCAPE_LIVE_LOGO_COLOR = `/static_vue/medscape-live-web/img/app-header-logo/mdsl-logo-color.png`
export const MEDSCAPE_LOGO = `/static_vue/medscape-live-web/img/medscape-logo.png`
export const MEDSCAPE_LIVE_LOGO_WHITE = `/static_vue/medscape-live-web/img/medscape-live-white.png`

export const MEDSCAPE_LIVE_REGISTRATION_CONFIRM = `/static_vue/medscape-live-web/img/icon-registered.png`

export const TWITTER = `/static_vue/medscape-live-web/img/twitter.svg`
export const YOUTUBE = `/static_vue/medscape-live-web/img/youtube.svg`
export const LINKEDIN = `/static_vue/medscape-live-web/img/linkedIn.svg`
export const FACEBOOK = `/static_vue/medscape-live-web/img/facebook.svg`
export const INSTAGRAM = `/static_vue/medscape-live-web/img/instagram.svg`

export const ABOUT_HEADER_BACKGROUND = `/static_vue/medscape-live-web/img/about-header-background/header-bg.png`
export const ABOUT_HEADER_BACKGROUND_SRCSET = `/static_vue/medscape-live-web/img/about-header-background/header-bg.png, /static_vue/medscape-live-web/img/about-header-background/header-bg@2x.png 2x, /static_vue/medscape-live-web/img/about-header-background/header-bg@3x.png 3x`

export const ABOUT_VIDEO_FILE = `/static_vue/medscape-live-web/video/MQ03_VirtualSymposiaSizzle_061120.mp4`
export const ABOUT_AUDIENCE_IMAGE = `/static_vue/medscape-live-web/img/about-audience-info.jpg`
export const ABOUT_VIDEO_POSTER = `/static_vue/medscape-live-web/img/about-video-poster.png`

export const ABOUT_MARKETING_CARD1 = `/static_vue/medscape-live-web/img/about-b2b-marketing/card1.png`
export const ABOUT_MARKETING_CARD2 = `/static_vue/medscape-live-web/img/about-b2b-marketing/card2.jpg`
export const ABOUT_MARKETING_CARD3 = `/static_vue/medscape-live-web/img/about-b2b-marketing/card3.jpg`
export const ABOUT_MARKETING_CARD4 = `/static_vue/medscape-live-web/img/about-b2b-marketing/card4.jpg`

export const DROP_DOWN_INDICATOR = `/static_vue/medscape-live-web/img/dropdown-indicator.svg`
export const RIGHT_INDICATOR = `/static_vue/medscape-live-web/img/right-indicator.svg`
export const LEFT_INDICATOR = `/static_vue/medscape-live-web/img/left-indicator.svg`

export const EMAIL_FORM_IMAGE = `/static_vue/medscape-live-web/img/email-form.svg`
export const USER_NOT_FOUND_ICON = `/static_vue/medscape-live-web/img/user-not-found-icon.svg`

export const REGISTER_UNAVAILABLE_IMAGE = `/static_vue/medscape-live-web/img/registration_unavailable.png`
export const REGISTER_ERROR_CLOSE = `/static_vue/medscape-live-web/img/error_close.png`

export const GOOGLE_CALENDAR_IMAGE = `/static_vue/medscape-live-web/img/google-calendar.png`
export const OUTLOOK_CALENDAR_IMAGE = `/static_vue/medscape-live-web/img/outlook-calendar.png`
export const YAHOO_CALENDAR_IMAGE = `/static_vue/medscape-live-web/img/yahoo-calendar.png`
export const ICS_CALENDAR_IMAGE = `/static_vue/medscape-live-web/img/ics-calendar.png`

export const LIVE_EVENT_LOCATION_ICON = `/static_vue/medscape-live-web/img/icon-pin.svg`
export const LIVE_EVENT_CLOCK_ICON = `/static_vue/medscape-live-web/img/icon-clock.svg`
export const LIVE_EVENT_SPECIALTY_ICON = `/static_vue/medscape-live-web/img/icon-specialty.svg`
export const NEED_HELP_ICON = `/static_vue/medscape-live-web/icons/Icon-awesome-question-circle.png`

export const MEDSCAPE_LIVE_REGISTRATION_CONFIRM_ENVELOPE = `/static_vue/medscape-live-web/img/registration-confirmation/envelope.png`

export const MEDSCAPE_LIVE_APP_DOWNLOAD_QR_CODE = `/static_vue/medscape-live-web/img/medscapelive-app/medscapelive-app-download-qr-code.png`
export const LIVE_EVENT_CIRCLE_CHECK = `/static_vue/medscape-live-web/img/icon-circle-check.svg`

export const HYBRID_CHEVRON = `/static_vue/medscape-live-web/img/hybrid-dialog-chevron-left.svg`
export const HYBRID_VIRTUAL = `/static_vue/medscape-live-web/img/hybrid-dialog-at-home.svg`
export const HYBRID_IN_PERSON = `/static_vue/medscape-live-web/img/hybrid-dialog-in-person.svg`
