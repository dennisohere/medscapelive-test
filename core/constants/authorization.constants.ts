export const DB_SERVICE_AUTH_USER_NAME = 'DBServiceUser'
export const GLOBAL_P2P_REGISTRATION_ERROR = '4003'
