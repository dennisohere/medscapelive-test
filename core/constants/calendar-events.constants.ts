export const MS_IN_MINUTES = 60 * 1000
export const DEFAULT_DURATION_MINUTES = 60

export const GOOGLE_PARAMS = {
  BASE_URL: 'https://calendar.google.com/calendar/render?action=TEMPLATE',
  QUERY_DATES: '&dates=',
  QUERY_FRONT_SLASH: '/',
  QUERY_LOCATION: '&location=',
  QUERY_TEXT: '&text='
}

export const OUTLOOK_PARAMS = {
  BASE_URL: 'https://outlook.live.com/calendar/0/action/compose?',
  QUERY_START_DATE: 'startdt=',
  QUERY_SUBJECT: '&subject=',
  QUERY_END_DATE: '&enddt=',
  QUERY_LOCATION: '&location='
}

export const YAHOO_PARAMS = {
  BASE_URL: 'https://calendar.yahoo.com/?v=60',
  QUERY_END_DATE: '&ET=',
  QUERY_LOCATION: '&in_loc=',
  QUERY_START_DATE: '&ST=',
  QUERY_TITLE: '&TITLE='
}

export const ICS_PARAMS = {
  BASE_URL: 'data:text/calendar;charset=utf8,',
  BEGIN_CAL: 'BEGIN:VCALENDAR',
  VERSION: 'VERSION:2.0',
  BEGIN_EVENT: 'BEGIN:VEVENT',
  QUERY_START_DATE: 'DTSTART:',
  QUERY_END_DATE: 'DTEND:',
  QUERY_SUMMARY: 'SUMMARY:',
  QUERY_LOCATION: 'LOCATION:',
  QUERY_END_EVENT: 'END:VEVENT',
  END_CAL: 'END:VCALENDAR'
}
export const IE_CALENDAR_PARAMS = {
  CALENDAR_MIME: 'text/calendar',
  ICS_DOWNLOAD_FILE_NAME: 'download.ics',
  IE_10_UA_FRAGMENT: 'MSIE ',
  IE_11_UA_FRAGMENT: 'Trident/'
}
