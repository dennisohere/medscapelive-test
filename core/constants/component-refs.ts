export const SEARCH_FILTER_SCROLL_TO_REF = 'searchFilterScrollTo'
export const SPECIALTY_SELECT_REF = 'specialtySelect'
export const LOCATION_SELECT_REF = 'locationSelect'
export const DATE_SELECT_REF = 'dateSelect'
export const HOME_SEARCH_HEADER = 'homeSearchHeader'
export const ABOUT_CONTACT_HEADER = 'aboutContactHeader'
export const VIDEO_PLAYER = 'videoPlayer'
export const EVENTS_HEADER = 'eventsHeader'
export const SEARCH_PANEL = 'searchPanel'
