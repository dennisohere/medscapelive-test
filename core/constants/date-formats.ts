export const YYYY_MM_DD = 'YYYY-MM-DD'
export const MM_DD_YYYY = 'MM-DD-YYYY'
