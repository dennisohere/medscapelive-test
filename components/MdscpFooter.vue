<template>
  <div>
    <div class="mdscp-footer-top">
      <div class="mdscp-footer-top-main mdscp-footer-top-container">
        <div class="mdscp-footer-top-logo">
          <img
            class="footer-image"
            :src="MEDSCAPE_LOGO"
            title="medscape logo"
            alt="logo of medscape"
          />
        </div>
      </div>
      <div class="mdscp-footer-top-main mdscp-footer-top-container">
        <div class="mdscp-footer-top-social">
          <h4>Find Us On</h4>
          <a
            v-for="(item, index) in footerMenu.social"
            :key="index"
            :href="item.path"
            target="_self"
            :alt="item.alt"
          >
            <img :src="item.imagePath" :title="item.title" :alt="item.alt" />
          </a>
        </div>
        <div class="mdscp-footer-top-section">
          <div class="mdscp-footer-top-col">
            <h4>About</h4>
            <template v-for="(item, index) in footerMenu.About">
              <a
                v-if="item.name !== 'gdpr-ccpa'"
                :key="index"
                :href="item.path"
                :target="item.target"
              >
                {{ item.name }}
              </a>
              <a
                v-else-if="item.name === 'gdpr-ccpa' && geoc === 'US'"
                id="ot-sdk-btn"
                :key="index + 10"
                class="ib-your-choices"
                onclick="OneTrust.ToggleInfoDisplay();"
                href="javascript:void(0);"
                style="display: none; color: #4b4b4b !important; font-size: .875rem !important; margin: 0 0 14px 0 !important;"
                >Your Privacy Choices
                <img
                  src="https://icons.internetbrands.com/ccpa/privacyoptions29x14.png"
                  style="vertical-align:middle;"
              /></a>
              <a
                v-else
                id="ot-sdk-btn"
                :key="index + 11"
                class="ot-sdk-show-settings"
                style="display: block;"
              >
              </a>
            </template>
            <div id="teconsent" class="mt-3"></div>
          </div>
          <div class="mdscp-footer-top-col">
            <h4>Membership</h4>
            <a
              v-for="(item, index) in footerMenu.Membership"
              :key="index"
              :href="item.path"
              :target="item.target"
            >
              {{ item.name }}
            </a>
          </div>
          <div class="mdscp-footer-top-col">
            <h4>
              <a href="http://www.medscape.com/public/applanding">
                App
              </a>
            </h4>
            <a
              v-for="(item, index) in footerMenu.Apps"
              :key="index"
              :href="item.path"
              :target="item.target"
            >
              {{ item.name }}
            </a>
          </div>
          <div class="mdscp-footer-top-col" style="display: none;">
            <h4>
              <span class="text-transform-none">
                WebMD Network
              </span>
            </h4>
            <a
              v-for="(item, index) in footerMenu.Network"
              :key="index"
              :href="item.path"
              :target="item.target"
            >
              {{ item.name }}
            </a>
          </div>
          <div class="mdscp-footer-top-col">
            <h4>Editions</h4>
            <a
              v-for="(item, index) in langMenu"
              :key="index"
              :href="item.path"
              :target="item.target"
            >
              {{ item.name }}
            </a>
          </div>
        </div>
      </div>
    </div>
    <div class="mdscp-footer-btm">
      <div class="mdscp-footer-btm-legal mdscp-footer-top-container">
        {{ copyright }}
      </div>
    </div>
  </div>
</template>
<script>
import { MEDSCAPE_LOGO } from '@/core/constants/asset-paths'

export default {
  name: 'Footer',
  props: {
    footerMenu: {
      type: Object,
      default() {
        return {}
      }
    },
    copyright: {
      type: String,
      default: ''
    },
    langMenu: {
      type: Array,
      default() {
        return {}
      }
    },
    formlang: {
      type: String,
      default: ''
    }
  },
  data() {
    return {
      MEDSCAPE_LOGO,
      activeTab: 0,
      geoc: ''
    }
  },
  mounted() {
    this.geoc = window?.PageMetadata?.reqHeaders?.geoc

    window.addEventListener('DOMContentLoaded', function() {
      if (document.getElementById('ot-stub') !== null) {
        const elem = document.getElementById('ot-sdk-btn')
        elem.style.setProperty('display', 'block', 'important')
      }
    })
  }
}
</script>
<style lang="scss">
$proxima_nova_ltlight: 'ProximaNova Light Italic', sans-serif;
$proximaFont: 'ProximaNova', sans-serif;

.mdscp-footer-top {
  border-top: solid 2px #005b81;
  padding-top: 2.5rem;
  padding-bottom: 4rem;
  background: #fff;
  @media (max-width: 767px) {
    padding-bottom: 0;
  }
  &-main {
    justify-content: space-between;
    flex-wrap: wrap;
  }
  &-container {
    max-width: 96%;
    width: 1240px;
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    justify-content: center;
    margin-left: auto;
    margin-right: auto;
    @media (max-width: 767px) {
      max-width: 90%;
    }
  }
  &-logo {
    width: 100%;
    text-align: left;
    margin-bottom: 1rem;
    @media (max-width: 767px) {
      margin-bottom: 0.5rem;
    }
    .footer-image {
      height: 1.75rem;
    }
  }
  &-section {
    display: flex;
    padding-bottom: 3rem;
    justify-content: space-between;
    @media (min-width: 768px) {
      flex: 5;
    }
    @media (max-width: 1023px) {
      justify-content: flex-start;
      flex-wrap: wrap;
    }
  }
  &-social {
    text-align: left;
    flex: 1.6;
    @media (max-width: 767px) {
      width: 100%;
      padding-bottom: 2rem;
    }
    a {
      margin-right: 0.5rem;
      margin-bottom: 0.5rem;
      display: inline-block;
    }
  }
  &-col {
    padding: 0 1rem;
    border-left: solid 1px #dbdbdb;
    font-family: $proximaFont;
    text-align: left;
    font-size: 0.875rem;
    span {
      display: block;
      margin-bottom: 0.875rem;
      color: #222;
      text-decoration: none;
      cursor: pointer;
    }
    @media (max-width: 767px) {
      margin-bottom: 0;
      padding: 0 0 2rem;
      border-left: none;
      border-top: solid 1px #dbdbdb;
      width: 100%;
    }
    @media (max-width: 1023px) {
      min-width: 9.75rem;
      margin-bottom: 1rem;
      padding-bottom: 1rem;
    }
    a {
      display: block;
      margin-bottom: 0.875rem;
      color: #222;
      text-decoration: none;
      &:last-of-type {
        margin-bottom: 0;
      }
      &:hover,
      &:active {
        color: #005b81;
      }
    }
  }
  h4 {
    margin-top: 0;
    text-transform: uppercase;
    font-size: 1rem;
    margin-bottom: 0.875rem;
    @media (max-width: 767px) {
      margin-top: 0.875rem;
    }
  }
}

.mdscp-footer-btm {
  background: #fff;
  &-legal {
    background: #fff;
    font-family: $proximaFont;
    width: 500px;
    margin-top: 0rem;
    margin-left: auto;
    margin-right: auto;
    font-size: 0.75rem;
    line-height: 1.3;
    text-align: center;
    color: #767674;
    padding-bottom: 2.5rem;
  }
}
</style>
