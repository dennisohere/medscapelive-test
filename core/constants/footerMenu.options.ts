import { FACEBOOK, TWITTER, LINKEDIN, YOUTUBE, INSTAGRAM } from './asset-paths'

export const footerMenu = {
  social: [
    {
      path: 'https://www.facebook.com/medscape',
      alt: 'Facebook',
      title: 'facebook icon',
      imagePath: FACEBOOK
    },
    {
      path: 'https://twitter.com/medscapelive',
      alt: 'Twitter',
      title: 'twitter icon',
      imagePath: TWITTER
    },
    {
      path: 'https://www.linkedin.com/showcase/medscapelive/',
      alt: 'LinkedIn',
      title: 'linkedin icon',
      imagePath: LINKEDIN
    },
    {
      path: 'https://www.youtube.com/user/medscape/',
      alt: 'Youtube',
      title: 'youtube icon',
      imagePath: YOUTUBE
    },
    {
      path: 'https://www.instagram.com/medscape/',
      alt: 'Instagram',
      title: 'instagram icon',
      imagePath: INSTAGRAM
    }
  ],
  About: [
    {
      path: 'https://www.medscape.com/public/about',
      name: 'About Medscape'
    },
    {
      path: 'https://www.medscape.com/public/privacy',
      name: 'Privacy Policy'
    },
    {
      path: 'https://www.medscape.com/public/editorialpolicies',
      name: 'Editorial Policy'
    },
    {
      path: 'https://www.medscape.com/public/cookies',
      name: 'Cookies'
    },
    {
      path: '',
      name: 'gdpr-ccpa'
    },
    {
      path: 'https://www.medscape.com/public/termsofuse',
      name: 'Terms of Use'
    },
    {
      path: 'https://www.medscape.com/public/adpolicy',
      name: 'Advertising Policy'
    },
    {
      path: 'http://help.medscape.com/',
      name: 'Help Center'
    }
  ],
  Membership: [
    {
      path: 'https://profreg.medscape.com/px/registration.do?lang=en',
      name: 'Become a Member'
    },
    {
      path: 'https://www.medscape.com/profile',
      name: 'About You'
    },
    {
      path: 'https://www.medscape.com/profile/professional',
      name: 'Professional Information'
    },
    {
      path: 'https://www.medscape.com/newsletters',
      name: 'Newsletters & Alerts'
    },
    {
      path: 'https://www.medscape.com/marketresearch',
      name: 'Market Research'
    }
  ],
  Apps: [
    {
      path: 'http://www.medscape.com/public/medscapeapp',
      name: 'Medscape'
    }
  ],
  Network: [
    {
      path: 'https://www.medscapelive.com/',
      name: 'Medscape Live Events'
    },
    {
      path: 'http://www.webmd.com/',
      name: 'WebMD'
    },
    {
      path: 'http://www.medicinenet.com/',
      name: 'MedicineNet'
    },
    {
      path: 'http://www.emedicinehealth.com/',
      name: 'eMedicineHealth'
    },
    {
      path: 'http://www.rxlist.com/',
      name: 'RxList'
    },
    {
      path: 'https://www.webmd.com/corporate/default.htm',
      name: 'WebMD Corporate'
    },
    {
      path: 'https://www.medscape.co.uk/',
      name: 'Medscape UK'
    }
  ]
}
export const langMenu = [
  {
    path: 'https://www.medscape.com/',
    name: 'English'
  },
  {
    path: 'http://deutsch.medscape.com/',
    name: 'Deutsch'
  },
  {
    path: 'http://espanol.medscape.com/',
    name: 'Español'
  },
  {
    path: 'http://francais.medscape.com/',
    name: 'Français'
  },
  {
    path: 'http://portugues.medscape.com/',
    name: 'Português'
  },
  {
    path: 'https://www.medscape.co.uk/',
    name: 'UK'
  }
]
export const copyright =
  'All material on this website is protected by copyright, Copyright © 1994-2023 by WebMD LLC. This website also contains material copyrighted by 3rd parties.'
