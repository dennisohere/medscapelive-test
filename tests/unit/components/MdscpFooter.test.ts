import {VueWrapper} from '@vue/test-utils'
import MdscpFooter from '@/components/MdscpFooter.vue'
import { FACEBOOK, TWITTER } from '~/core/constants/asset-paths'

import { describe, it, expect, beforeEach } from 'vitest'
import {createNuxtTestWrapper} from "~/tests/unit/setup/create-wrapper";


describe('MdscpFooter.vue', () => {
  let wrapper: VueWrapper
  beforeEach(() => {
    wrapper = createNuxtTestWrapper(MdscpFooter, {
      propsData: {
        footerMenu: {
          social: [
            {
              path: 'https://www.facebook.com/medscape',
              alt: 'Facebook',
              imagePath: FACEBOOK,
              title: 'facebook icon'
            },
            {
              path: 'https://twitter.com/medscapelive',
              alt: 'Twitter',
              imagePath: TWITTER,
              title: 'twitter icon'
            }
          ],
          About: [
            {
              path: 'https://www.medscape.com/public/privacy',
              name: 'Privacy Policy'
            },
            {
              path: 'https://www.medscape.com/public/cookies',
              name: 'Cookies'
            }
          ],
          Membership: [
            {
              path: 'https://profreg.medscape.com/px/registration.do?lang=en',
              name: 'Become a Member'
            }
          ],
          Apps: [
            {
              path: 'http://www.medscape.com/public/medscapeapp',
              name: 'Medscape'
            }
          ],
          Network: [
            {
              path: 'http://www.webmd.com/',
              name: 'WebMD'
            }
          ]
        },
        copyright:
          'All material on this website is protected by copyright, Copyright Â© 1994-2022 by WebMD LLC. This website also contains material copyrighted by 3rd parties.',
        langMenu: [
          {
            path: 'http://www.webmd.com/',
            name: 'English'
          },
          {
            path: 'http://www.deutsch.webmd.com/',
            name: 'Deutsch'
          }
        ]
      }
    })
  })
  describe('render', () => {
    it('matches the snapshot', () => {
      // Assert
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
