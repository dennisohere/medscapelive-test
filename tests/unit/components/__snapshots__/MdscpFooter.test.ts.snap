// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdscpFooter.vue > render > matches the snapshot 1`] = `
<div>
  <div
    class="mdscp-footer-top"
  >
    <div
      class="mdscp-footer-top-main mdscp-footer-top-container"
    >
      <div
        class="mdscp-footer-top-logo"
      >
        <img
          alt="logo of medscape"
          class="footer-image"
          src="/static_vue/medscape-live-web/img/medscape-logo.png"
          title="medscape logo"
        />
      </div>
    </div>
    <div
      class="mdscp-footer-top-main mdscp-footer-top-container"
    >
      <div
        class="mdscp-footer-top-social"
      >
        <h4>
          Find Us On
        </h4>
        
        <a
          alt="Facebook"
          href="https://www.facebook.com/medscape"
          target="_self"
        >
          <img
            alt="Facebook"
            src="/static_vue/medscape-live-web/img/facebook.svg"
            title="facebook icon"
          />
        </a>
        <a
          alt="Twitter"
          href="https://twitter.com/medscapelive"
          target="_self"
        >
          <img
            alt="Twitter"
            src="/static_vue/medscape-live-web/img/twitter.svg"
            title="twitter icon"
          />
        </a>
        
      </div>
      <div
        class="mdscp-footer-top-section"
      >
        <div
          class="mdscp-footer-top-col"
        >
          <h4>
            About
          </h4>
          
          
          <a
            href="https://www.medscape.com/public/privacy"
          >
            Privacy Policy
          </a>
          
          
          <a
            href="https://www.medscape.com/public/cookies"
          >
            Cookies
          </a>
          
          
          <div
            class="mt-3"
            id="teconsent"
          />
        </div>
        <div
          class="mdscp-footer-top-col"
        >
          <h4>
            Membership
          </h4>
          
          <a
            href="https://profreg.medscape.com/px/registration.do?lang=en"
          >
            Become a Member
          </a>
          
        </div>
        <div
          class="mdscp-footer-top-col"
        >
          <h4>
            <a
              href="http://www.medscape.com/public/applanding"
            >
               App 
            </a>
          </h4>
          
          <a
            href="http://www.medscape.com/public/medscapeapp"
          >
            Medscape
          </a>
          
        </div>
        <div
          class="mdscp-footer-top-col"
          style="display: none;"
        >
          <h4>
            <span
              class="text-transform-none"
            >
               WebMD Network 
            </span>
          </h4>
          
          <a
            href="http://www.webmd.com/"
          >
            WebMD
          </a>
          
        </div>
        <div
          class="mdscp-footer-top-col"
        >
          <h4>
            Editions
          </h4>
          
          <a
            href="http://www.webmd.com/"
          >
            English
          </a>
          <a
            href="http://www.deutsch.webmd.com/"
          >
            Deutsch
          </a>
          
        </div>
      </div>
    </div>
  </div>
  <div
    class="mdscp-footer-btm"
  >
    <div
      class="mdscp-footer-btm-legal mdscp-footer-top-container"
    >
      All material on this website is protected by copyright, Copyright Â© 1994-2022 by WebMD LLC. This website also contains material copyrighted by 3rd parties.
    </div>
  </div>
</div>
`;
